test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -4334,6 +4476,274 @@ def test_cgroup_default_config(self):
+                                    enforce_flags=('true', 'true'),
+                                    exclhost=False):
+        if not self.mem:
+            self.skipTest('Test requires memory subystem mounted')
+        if not self.swapctl:
+            self.skipTest('Test requires memsw accounting enabled')
+
+        self.load_config(self.cfg16
+                         % enforce_flags)
+        # Restart mom for changes made by cgroups hook to take effect
+        begin = time.time()
+        # sleep 2s to ensure log matching will not catch older log lines
+        time.sleep(2)
+        self.mom.restart()
+        # Make sure the MoM is restarted
+        self.mom.log_match('Hook handler returned success'
+                           ' for exechost_startup',
+                           starttime=begin, existence=True,
+                           interval=1, tail=False)
+        self.server.expect(NODE, {'state': 'free'},
+                           id=self.mom.shortname, interval=2, offset=1)
+
+        a = {'Resource_List.select':
+             '1:ncpus=1:vnode=%s'
+             % self.mom.shortname}
+        if exclhost:
+            a['Resource_List.place'] = 'exclhost'
+
+        j = Job(TEST_USER, attrs=a)
+        j.create_script(self.sleep100_job)
+        jid = self.server.submit(j)
+        bs = {'job_state': 'R'}
+        self.server.expect(JOB, bs, jid, offset=1)
+
+        mem_base = os.path.join(self.paths[self.hosts_list[0]]['memory'],
+                                'pbs_jobs.service', 'jobid')
+
+        # Get total physical memory available
+        mem_avail = os.path.join(mem_base,
+                                 'memory.limit_in_bytes')
+        result = self.du.cat(hostname=self.mom.hostname, filename=mem_avail,
+                             sudo=True)
+        mem_avail_in_bytes = None
+        try:
+            mem_avail_in_bytes = int(result['out'][0])
+        except Exception:
+            # None will be seen as a failure, nothing to do
+            pass
+        self.logger.info("total available mem: %d"
+                         % mem_avail_in_bytes)
+        self.assertTrue(mem_avail_in_bytes is not None,
+                        "Unable to read total memsw available")
+
+        # Get total phys+swap memory available
+        vmem_avail = os.path.join(mem_base,
+                                  'memory.memsw.limit_in_bytes')
+        result = self.du.cat(hostname=self.mom.hostname, filename=vmem_avail,
+                             sudo=True)
+        vmem_avail_in_bytes = None
+        try:
+            vmem_avail_in_bytes = int(result['out'][0])
+        except Exception:
+            # None will be seen as a failure, nothing to do
+            pass
+        self.logger.info("total available memsw: %d"
+                         % vmem_avail_in_bytes)
+        self.assertTrue(vmem_avail_in_bytes is not None,
+                        "Unable to read total memsw available")
```

Comment:
Can you make this error assertion unique to make it easier to tell which part is failing?  Right now it matches mem_avail_in_bytes.

Reply:
?

Reply:
Yeah, there is a typo in the earlier message.